### PR TITLE
Cspell refactor

### PR DESCRIPTION
**Pull request by FirelightFlagboy**

### CODE REVIEW

File: server/tests/run_parallel.py [hunk 1]
```diff
@@ -50,27 +56,27 @@ def sendevent(self, name, **kwargs):
...
+        self.sendevent(EVT_WORKER_FINISHED, exitstatus=exitstatus)
 
     @pytest.hookimpl
     def pytest_runtest_logreport(self, report: pytest.TestReport):
         # Ignore tests skipped for being out of the job's tests slice
         if ("test_out_of_slice", True) in report.user_properties:
             return
         self.sendevent(
-            "testreport",
+            EVT_TEST_REPORT,
```
Comment (mmmarcos) [inline server/tests/run_parallel.py:79]: This value is being changed (here and also below). Can't this have an impact elsewhere?

**Review by mmmarcos — CHANGES_REQUESTED** [review]

File: .cspell/names.txt [hunk 2]
```diff
@@ -8,34 +8,42 @@ bkryza
...
 cunyet
 Dalvik
 dorny
 dtolnay
 Dupre
+execnet
 expiredorg
 farn
 fifi
+Flagboy
```
[the remark this answers is not in the document]
Comment (FirelightFlagboy) [reply .cspell/names.txt:23]: in `release.yml`

File: server/tests/run_parallel.py [hunk 3]
```diff
@@ -50,27 +56,27 @@ def sendevent(self, name, **kwargs):
...
 
     @pytest.hookimpl
     def pytest_sessionstart(self, session):
-        self.sendevent("workerready")
+        self.sendevent(EVT_WORKER_READY)
 
     @pytest.hookimpl
     def pytest_sessionfinish(self, exitstatus):
-        self.sendevent("workerfinished", exitstatus=exitstatus)
+        self.sendevent(EVT_WORKER_FINISHED, exitstatus=exitstatus)
```
[the remark this answers is not in the document]
Comment (FirelightFlagboy) [reply server/tests/run_parallel.py:71]: All the place that were using that value now use a constant value

File: server/tests/run_parallel.py (hunk 1 above)
Comment (FirelightFlagboy) [reply server/tests/run_parallel.py:79]: https://github.com/Scille/parsec-cloud/pull/5131#discussion_r1317278886

File: server/tests/run_parallel.py [hunk 4]
```diff
@@ -50,27 +56,27 @@ def sendevent(self, name, **kwargs):
...
 
     @pytest.hookimpl
     def pytest_collection_finish(self, session: pytest.Session):
-        self.sendevent("collection_finish", total_tests_count=len(session.items))
+        self.sendevent(EVT_COLLECTION_FINISH, total_tests_count=len(session.items))
 
     @pytest.hookimpl
     def pytest_sessionstart(self, session):
-        self.sendevent("workerready")
+        self.sendevent(EVT_WORKER_READY)
```
[the remark this answers is not in the document]
Comment (FirelightFlagboy) [reply server/tests/run_parallel.py:67]: https://github.com/Scille/parsec-cloud/pull/5131#discussion_r1317278886

File: .cspell/custom-words.txt [hunk 5]
```diff
@@ -353,12 +458,16 @@ retrier
...
+ROOTDIR
+ROOTDIRS
+ROOTDIRSRAW
 RSAES
 RSASSA
 RsTest
 rungui
 runmountpoint
 runslow
+ruscrypto
```
[the remark this answers is not in the document]
Comment (FirelightFlagboy) [reply .cspell/custom-words.txt:470]: done

File: .cspell/custom-words.txt [hunk 6]
```diff
@@ -334,16 +435,20 @@ qrichtext
 qtbot
 qthelp
 qtrio
+quickstart
 Quotify
 randombytes
 randr
+readthedoc
```
[the remark this answers is not in the document]
Comment (FirelightFlagboy) [reply .cspell/custom-words.txt:442]: done

**Review by mmmarcos — APPROVED** [review]
:+1:

**Review by touilleMan — APPROVED** [review]

**Pull request merged**